src/hermite/cubic.jl

```diff
@@ -0,0 +1,163 @@
+struct CubicHermite
+    xs::Vector{Float64}
+    ys::Vector{Float64}
+    dydxs::Vector{Float64}
+    function CubicHermite(xs, ys, dydxs)
+        if length(xs) < 2
+            throw(
+                DomainError(
+                    "There must be at least two samples in the domain of the CubicHermite interpolator.",
+                ),
```

Comment:
Test is needed here to check this error is thrown.

Reply:
Fixed.